src/BugsnagUnity/Native/Android/NativeInterface.cs

```diff
@@ -853,7 +853,7 @@ private AndroidJavaObject BuildJavaStringDisposable(string input)
             }
         }
 
-        private bool IsUnity2019OrNewer()
+        internal static bool IsUnity2019OrNewer()
```

Comment:
Do these definitely need to be static? If it's unavoidable then we can go down this route, but it feels like it'd be preferable to keep things non-static (this will make it much easier to pass in a mock NativeInterface to other classes etc)

Reply:
I think this is ok for now as i don't want to duplicate code